Sources/Core/Other/WindowMonitor.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  WindowMonitor.swift
```

Comment:
Please make the header comment match other header comments exactly. For example, line 3 is incorrect, date should be ISO, copyright year should be 2018.